src/pages/submission.tsx

```diff
@@ -1,17 +1,63 @@
-/* eslint-disable no-unused-vars */
 import * as React from "react";
-import { Link } from "gatsby";
-
 import Layout from "../components/layout";
-import SEO from "../components/seo";
+import { Typography, Row, Col, Card } from "antd";
+import { number, string } from "prop-types";
+
+const submissionPageCards = [
+	{
+		number: 1,
+		title: "Download the template",
+		text:
+			"Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit."
+	},
+	{
+		number: 2,
+		title: "Add your data",
+		text:
+			"Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit"
+	},
+	{
+		number: 3,
+		title: "Upload your data to a repository",
+		text:
+			"Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit"
+	}
+];
+interface Card {
+	number: number;
+	title: string;
+	text: string;
+}
+interface SubmissionCardsProps {
+	cards: { card };
```

Comment:
This doesn't match the usage on line 35 and should reference the interface for Card.

Typescript should have been checking this for you there may be something wrong with the TS configuation. I'll investigate